keel-web/src/main/kotlin/com/netflix/spinnaker/config/KeelConfigurationFinalizer.kt

```diff
@@ -34,6 +33,8 @@ class KeelConfigurationFinalizer(
 
   private val log by lazy { LoggerFactory.getLogger(javaClass) }
 
+  private val kinds: List<SupportedKind<*>> by lazy { resourceHandlers.map { it.supportedKind } }
```

Comment:
Shouldn't we get this from the `ExtensionRegistry`?

Reply:
We can. It's just used to print the list of kinds at the very end, and in this class in particular the registry is populated off the same list of resource handlers, so I didn't bother.

Reply:
Yeah, it's not a big deal. I just feel like if we want to treat `ExtensionRegistry` as the source of truth for this kind of thing, we might as well use it everywhere. Avoids people copy 'n pasting this if nothing else.